src/components/learning-guide.cjsx

```diff
@@ -1,21 +1,32 @@
 React = require 'react'
 BS = require 'react-bootstrap'
+Router = require 'react-router'
+_ = require 'underscore'
 
+{CourseActions, CourseStore} = require '../flux/course'
 {LearningGuideStore, LearningGuideActions} = require '../flux/learning-guide'
 LoadableItem = require './loadable-item'
 LearningGuideChart = require './learning-guide-chart'
+PracticeButton = require './practice-button'
 
 
 LearningGuide = React.createClass
   displayName: 'LearningGuide'
+  contextTypes:
+    router: React.PropTypes.func
+
   propTypes:
     courseId: React.PropTypes.any.isRequired
 
+  getInitialState: ->
+    unit: false
+
   navigateToPractice: (unit) ->
-    alert "TODO: Navigate to practice unit ID #{unit.id} (#{unit.title})"
+    {courseId} = @props
+    @context.router.transitionTo('viewPractice', {courseId})
```

Comment:
I think this may need to create a new practice in addition to `transitionTo` and send the tags (in the unit) to call it complete. But that can be a separate PR